CommonTools/PileupAlgos/plugins/PuppiPhoton.cc

```diff
@@ -76,7 +71,25 @@ void PuppiPhoton::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
   edm::Handle<CandidateView> hPuppiProduct;
   iEvent.getByToken(tokenPuppiCandidates_,hPuppiProduct);
   const CandidateView *pupCol = hPuppiProduct.product();
-  for(CandidateView::const_iterator itPho = phoCol->begin(); itPho!=phoCol->end(); itPho++) {
+  if(usePFphotons_) {
+   for(CandidateView::const_iterator itPho = pfCol->begin(); itPho!=pfCol->end(); itPho++) {
```

Comment:
this gives me a
"/afs/cern.ch/user/h/hinzmann/stable_13TeV/jetmet/CMSSW_9_3_0_pre1/src/CommonTools/PileupAlgos/plugins/PuppiPhoton.cc: In member function 'virtual void PuppiPhoton::produce(edm::Event&, const edm::EventSetup&)':
/afs/cern.ch/user/h/hinzmann/stable_13TeV/jetmet/CMSSW_9_3_0_pre1/src/CommonTools/PileupAlgos/plugins/PuppiPhoton.cc:75:27: error: 'begin' was not declared in this scope
    for(const auto & pho : pfCol) {
"
what's the exact syntax for
const CandidateView *pfCol = hPFProduct.product();